src/RPCClient.js

```diff
@@ -112,6 +112,50 @@ class RPCClient {
     })
   }
 
+  /**
+   * @param {String} action
+   * @param {*} data
+   * @param {Number} timeoutMs
+   * @return {Promise}
+   * */
+  callAction (action, data, timeoutMs) {
+    let channel
```

Comment:
As far as I can see this `callAction` is almost the exact copy of the `call` method.
I would try something like this:
```javascript
callAction(action, data, timeoutMs){
    return this.call({action, data}, timeoutMs)
}
```
I think this would create the same result...

Reply:
fix